src/invoke.cpp

```diff
@@ -3447,19 +3448,20 @@ static void fcall_sam_async( lk::invoke_t &cxt )
 	cxt.result().empty_hash();
 
 	lk_string fn = cxt.arg(0).as_string();
-	FILE *fp = fopen(fn.c_str(), "r");
-	if (!fp)
+	std::ifstream fp;
```

Comment:
Would have to be tested for any file access issues using the PVRPM script that uses these functions